resources/views/tasks/edit.blade.php

```diff
@@ -222,6 +222,7 @@ class="d-inline-flex pull-left align-items-center"
     };
 
     const task = @json($task);
+    console.log(task);
```

Comment:
@marcoAntonioNina ,
This console.log it's important? It's possible to remove it?

Reply:
fixedthis console.log was removed